File: ExpressDine_Server/src/main/java/com/cybage/service/OrderFeedbackService.java
package com.cybage.service;

import java.util.List;

import com.cybage.model.OrderFeedback;

public interface OrderFeedbackService {
	
	void addOrderFeedback(OrderFeedback orderFeedback);
	
	List<OrderFeedback> findOrderFeedback();
	
	OrderFeedback getOrderFeedbackById(int id);

	List<OrderFeedback> getByRestaurant(int ownerId);

}

File: expressdine-client/src/Services/OrderFeedbackService.js

import axios from "axios";

const ed_url = "http://localhost:8080/orderFeedback";

export class OrderFeedbackService{
  addOrderFeedback = (orderFeedback) => {
    return axios.post(ed_url + "/add", orderFeedback);
  };

  getOrderFeedbackByUser = (userId) => {
    return axios.get(ed_url + "/getByRestaurant/" + userId);
  };
}

export default new OrderFeedbackService();
